src/components/Oscillators.js

```diff
@@ -85,22 +85,18 @@ export default function Oscillators({
           >
             <Square />
           </button>{" "}
-          <button
-            id="sine"
-            className={"Change"}
-            onClick={handleChangeOsc2Type}
-          >
+          <button id="sine" className={"Change"} onClick={handleChangeOsc2Type}>
```

Comment:
Do you need the `{` and the `}` here? after `className`
```suggestion
          <button id="sine" className="Change" onClick={handleChangeOsc2Type}>
```

Reply:
I tried out both - I don't need the curly braces 😃